AAP2336 - Created topics for Migration & Upgrade of Operator on OCP (#460)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-post-migration-cleanup.adoc

[id="post-migration-cleanup_{context}"]

= Post migration cleanup

[role=_abstract]

> Once your data migration is complete, you need to delete any InstanceGroups that are no longer required.

Reviewer: Based on [IBM Word usage "once"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o) and [IBM Word usage "need to"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n).  
```suggestion
After your data migration is complete, you must delete any InstanceGroups that are no longer required.
```

Author: Fixed.
